app/upgrades/v16/upgrades.go

```diff
@@ -9,6 +9,22 @@ import (
 	"github.com/osmosis-labs/osmosis/v15/app/upgrades"
 )
 
+const (
+	// TODO: change this to OSMO / DAI pool ID
+	cfmmPoolIdToLink = uint64(1)
```

Comment:
can we link the osmo/dai pool here in the comment and just change the pool ID to that for now?

Reply:
yeah this is remaining TODO